cluster/engine.go

```diff
@@ -20,7 +21,9 @@ import (
 
 const (
 	// Force-refresh the state of the engine this often.
-	stateRefreshPeriod = 30 * time.Second
+	stateRefreshMinRange = 30
```

Comment:
We can tweak those values as needed. (And ideally be able to change those values when launching the Manager.)